packages/perseus/src/widgets/video.tsx

```diff
@@ -75,6 +76,16 @@ class Video extends React.Component<Props> {
 
         if (IS_URL.test(location)) {
             url = location;
+            if (IS_VIMEO.test(url)) {
+                // If this is a vimeo video, we need to add the query string
+                // parameter "dnt" so that analytics/tracking cookies aren't set.
+                // https://help.vimeo.com/hc/en-us/articles/12426260232977-Player-parameters-overview
+                if (url.indexOf("?") === -1) {
+                    url += "?dnt=1";
+                } else {
+                    url += "&dnt=1";
+                }
```

Comment:
No action needed: I think this is fine to ship as-is, but there's also the [URLSearchParams API](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) now.